hugo-modules/core/utils/asset/image/img.html

```diff
@@ -51,7 +51,7 @@
       {{- $width := .width -}}
       {{- $height := .height -}}
       {{/* Preload hint is added in sources.html */}}
-       <img src="{{- $src -}}" width="{{- $width -}}" height="{{- $height -}}" {{ $image_attributes }} alt="{{- $alt -}}"{{ with $params.preload }} fetchpriority="high"{{ end }}>
```

Comment:
why is this removed?

Reply:
I thought we needed it because with different ratios the sizes come from the `<source>` elements. Maybe it is enough there. But I just noticed another problem, there is a layout shift when the sources have a media attribute. I am looking for a solution.